src/goto-symex/builtin_functions.cpp

```diff
@@ -1704,3 +1618,26 @@ bool goto_symext::run_builtin(
 
   return false;
 }
+
+void goto_symext::replace_races_check(expr2tc &expr)
```

Comment:
Can you write some documentation here? You could reuse the text from this PR.

Reply:
done